src/Controller/PostNoteController.php

```diff
@@ -100,6 +100,7 @@ protected function defaultAction()
 
         if ($this->cat == 'post_result' && ($post_result = $get->getInt('post_result'))) {
             $this->tpl->assign('post_result', $post_result);
+            $this->tpl->assign('garlic_prefix', $get->has('garlic_prefix') ? $get->get('garlic_prefix') : '');
```

Comment:
if you don't have "0" there as valid value, you can write this as:

`$get->get('garlic_prefix', '')` to have default value empty string